content/seismic/refraction/seismic_velocity.rst

```diff
@@ -125,6 +125,8 @@ Finally even at depth, as the pore pressure increases above hydrostatic, the eff
 
 All this seems a bit complicated, and the take-home message is that there are several contributing factors to velocity, some of which may be counter-intuitive.
 
+.. _myreferencelabel:
```

Comment:
Could you make the reference label more descriptive / context specific? Since you are labeling the Fluid Saturation section, maybe `_seismic_velocity_fluid_saturation`